idemp-base/src/org/bandahealth/idempiere/base/model/MBandaSetup.java

```diff
@@ -1266,9 +1266,18 @@ public boolean openCalendarYearPeriods() {
 
 	public boolean createProductAttributeSets() {
 		// Get all active, attribute sets from the default configuration client
-		List<MAttributeSet> attributeSets = new Query(context, MAttributeSet.Table_Name,
+		// PO.setCrossTenantSafe();
```

Comment:
We need to upgrade to iDempiere 8+ to avoid commenting out such. If for instance a new dev is running on 8 and forgets to comment out this line, then I'd presume we'll have errors on PROD..

Reply:
Agreed on that. Upgrading has been on the backlog for a long time. If a new dev starts running on 8+, we can ask why they didn't just pull our code from the repo that is 7.1. And they'll run into much bigger issues with iDempiere 9, so I think we can just emphasize to pull our code from the repo.

After we get the testing done, we can tell them to just pull a docker image to run iDempiere, too.

Reply:
cool